scripts/learn_kind_images.helper.mk

```diff
@@ -33,6 +33,9 @@ bin_dir := _bin
 $(bin_dir) $(bin_dir)/scratch:
 	mkdir -p $@
 
+HOST_OS := unused
+HOST_ARCH := unused
```

Comment:
this fixes a warning message that the `make upgrade-kind-images` was outputting.